docs/userGuide/siteConfiguration.md

```diff
@@ -50,7 +50,14 @@ Here is a typical `site.json` file:
     "*.mbd",
     ".git/*"
   ],
-  "tags": ["tag1", "tag2"],
+  "plugins" : [
+    "filterTags"
+  ],
+  "pluginsContext" : {
+    "filterTags" : {
+      "tags": ["tag1", "tag2"]
+    }

```

Comment:
The addition is good, but there's a lack of explanation of the `plugins` and the `pluginsContext` attribute.

The `tags` explanation below will also seems to be out of place now (since we will be explaining plugin settings for all kinds of plugin). Maybe we can say something like "the example above uses `tags` as an example of configuring plugin settings, refer to the Tag plugin for more details" or something similar?

Reply:
Sure, I reworded it